setup.py

```diff
@@ -17,13 +17,12 @@
     license="GPLv3+",
     python_requires=">=3.5",
     url="https://github.com/freedomofpress/securdrop-workstation",
-    data_files=[("share/securedrop-workstation-dom0-config", ["config.json.example"])],
```

Comment:
will the removal of the data_file here have any implication in the docs? Since the docs say we should copy the file, I don't think there's an impact: https://workstation.securedrop.org/en/stable/admin/install.html?highlight=config.json.example#configure-securedrop-workstation-estimated-wait-time-60-90-minutes

Reply:
No implication: the `config.json.example` file should be installed to the exact same location as already documented.